.tool-versions

```diff
@@ -0,0 +1 @@
+ruby 2.4.4
```

Comment:
As a user of [asdf](https://github.com/asdf-vm/asdf), this helps ensure I'm using the right version of Ruby.